scripts/animatediff_output.py

```diff
@@ -278,9 +283,15 @@ def _save(
                     file.container_metadata["Title"] = infotext
                     file.container_metadata["Comment"] = infotext
                 file.write(video_array, codec="vp9", fps=params.fps)
-        
+
         if s3_enable:
-            for video_path in video_paths: self._save_to_s3_stroge(video_path)  
+            for video_path in video_paths: self._save_to_s3_stroge(video_path)
+
+        try:
```

Comment:
`try` without `except`?

Reply:
try is necessary, in case of callbacks failed.